client/my-sites/plugins/controller-logged-out.js

```diff
@@ -54,12 +91,16 @@ export async function fetchPlugins( context, next ) {
 		...getQueryOptions( context ),
 	};
 
-	await Promise.all( [
-		prefetchProductList( store ),
-		prefetchPaidPlugins( queryClient, options ),
-		prefetchPopularPlugins( queryClient, options ),
-		prefetchFeaturedPlugins( queryClient, options ),
-	] );
+	await prefetchTimebox(
+		[
+			prefetchProductList( queryClient, store ),
+			prefetchPaidPlugins( queryClient, options ),
+			prefetchPopularPlugins( queryClient, options ),
+			prefetchFeaturedPlugins( queryClient, options ),
+		],
+		context,
+		'fetchPlugins'
```

Comment:
I was just testing everything again, and the page works great with SSR disabled. However, I can't get the timeout to trigger, even with it set to 5ms.

I'm wondering if this is an issue though: we call `prefetchTimebox` with three arguments, the third being a string "fetchPlugins". However, in the function signature of `prefetchTimebox`, the third argument is `timeout`. We only set the timeout to our constants if that argument is undefined.

So I think on the line with `setTimeout( reject, timeout, new Error( PREFETCH_TIMEOUT_ERROR ) )`, the variable timeout is the string "fetchPlugins", since we never call `prefetchTimebox` with anything else. I might be missing something, but I think that's why the timeout doesn't trigger for me any more.

Reply:
Thanks for catching this. The 3rd paramter used to be the `key` property used for logging. I removed it now, since we should be able to determine where the error was triggered via the `path` or `url` props.

I remove the "fetchPlugins" and everything should work as before.